strings/find_anagram_indices.py

```diff
@@ -0,0 +1,81 @@
+from collections import Counter, defaultdict
+
+
+def find_anagram_indices(word: str, letters: str) -> list:
+    """
+    Given a word `word`, and a string `letters`,
+    find all indices in `letters` which are the starting
+    locations of anagrams of `word`.
+    For example, given `word` is `the` and
+    `letters` is `tehxetheht`, return `[0, 4, 5, 7]`.
+
+    1. Brute force solution
+
+    This algorithm would take O(w x s) time
+    - x is the length of the word
+    - y is the length of the input string
+
+    :param word: word to be analyzed
+    :param letters: complete word to be scanned
+    :return: Array
+
+    >>> find_anagram_indices("the", "tehxetheht")
+    [0, 4, 5, 7]
+    """
+    output = []
+    for i in range(len(letters) - len(word) + 1):
+        window = letters[i : i + len(word)]
+        if Counter(window) == Counter(word):
+            output.append(i)
+    return output
+
+
+def find_anagram_indices_2(word: str, letters: str) -> list:
```

Comment:
Please provide the generic parameters for the list type and use List[type] from the typing module, import it using from typing import List.

Also, the function name should be like `find_anagram_indices_efficient` to denote that this is the efficient version.